packages/devops/scripts/deployment/deployLatestRepositories.sh

```diff
@@ -36,16 +45,9 @@ for PACKAGE in "meditrak-server" "web-config-server" "web-frontend" "admin-panel
         jq -r '.Parameters| .[] | .Name + "=\"" + .Value + "\""  ' |
         sed -e "s~${SSM_PATH}/~~" >.env)
 
-    # If there were no environment variables for the specified branch, default to dev
-    if [ ! -s .env ]; then
-        echo "Checking out default dev environment variables"
-        SSM_PATH="/${PACKAGE}/dev"
-        $(aws ssm get-parameters-by-path --with-decryption --path $SSM_PATH |
-            jq -r '.Parameters| .[] | .Name + "=\"" + .Value + "\""  ' |
-            sed -e "s~${SSM_PATH}/~~" >.env)
-        # Replace any instances of "dev" in the .env file (e.g. to point urls in the right place)
-        sed -i -e "s/dev/${STAGE}/g" .env
-    fi
+    # Replace any instances of the placeholder [branch-name] in the .env file with the actual branch
+    # name (e.g. [branch-name]-api.tupaia.org -> specific-branch-api.tupaia.org)
+    sed -i -e "s/\[branch-name\]/${BRANCH}/g" .env
```

Comment:
Clever idea using a placeholder!! I guess this has been updated in AWS SSM, and we don't need to update our Last Pass `.env` files? Since if `[branch-name]` is not found, then no replacement will occur, which is fine in case of `[DEV]` and `[PROD]` .env files

Reply:
Updating them for dev will need to be part of the release process. See https://github.com/beyondessential/tupaia-backlog/issues/827#issuecomment-719098624